include/xtensor/xadapt.hpp

```diff
@@ -124,12 +133,21 @@ namespace xt
      * @param l the layout_type of the xarray_adaptor
      * @param alloc the allocator used for allocating / deallocating the dynamic array
      */
-    template <layout_type L = XTENSOR_DEFAULT_LAYOUT, class P, class O, class SC, class A = detail::default_allocator_for_ptr_t<P>,
+    template <layout_type L = XTENSOR_DEFAULT_LAYOUT,
+              class P,
+              class O,
+              class SC,
+              class A = detail::default_allocator_for_ptr_t<P>,
               XTL_REQUIRES(detail::not_an_array<std::decay_t<SC>>)>
-    inline xarray_adaptor<xbuffer_adaptor<xtl::closure_type_t<P>, O, A>, L, SC>
-    adapt(P&& pointer, typename A::size_type size, O ownership, const SC& shape, layout_type l = L, const A& alloc = A())
+    inline xarray_adaptor<xbuffer_adaptor<xtl::closure_type_t<P>, O, A>, L, SC> adapt(
```

Comment:
The function name should be on its own line, it's easier to see when scrolling in the file.